crypto/ecdh.c

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (C) 2020 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+#include "ecdh.h"
```

Comment:
Should it be `crypto/ecdh.h`?

Reply:
They are under the same directory tho.

Reply:
If `crypto/ecdh.h` is a public header, always use full path for consistency.

Reply:
got it